backend/cesium_api/__main__.py

```diff
@@ -4,7 +4,7 @@
 from .app import App
 from .data import Data
 from flask_cors import CORS
-from . import APP_NAME, APP_DESCRIPTION, APP_VERSION, SATELLITES, GROUND_STATIONS, DEFAULT_HOST, DEFAULT_PORT, DEFAULT_DATA_DIR, DEFAULT_API_PREFIX
+from . import APP_NAME, APP_DESCRIPTION, APP_VERSION, SATELLITES, GROUND_STATIONS, DEFAULT_HOST, DEFAULT_PORT, DEFAULT_DATA_DIR, DEFAULT_API_PREFIX, DEFAULT_TILES_DIR
```

Comment:
```suggestion
from . import APP_NAME, APP_DESCRIPTION, APP_VERSION, SATELLITES, GROUND_STATIONS, DEFAULT_HOST, DEFAULT_PORT, DEFAULT_DATA_DIR, DEFAULT_API_PREFIX
```

See related comment below about `DEFAULT_DATA_DIR` vs `DEFAULT_TILES_DIR`.

But make sure to remove this here.